sdk/setup.py

```diff
@@ -30,4 +30,6 @@
         'dev': dev_requirements
     },
     python_requires='>=3.7',
+    long_description=pathlib.Path('./docs/README.md').read_text(),
+    long_descreiption_context_type='text/markdown'
```

Comment:
I've added this in so that the PyPI release would show the `README.md` in the docs.

Reply:
Should this be `long_description_content_type` ?

Reply:
Whoops, thanks for spotting that; my spelling was atrocious xD